src/content/guides/author-libraries.md

```diff
@@ -99,15 +100,15 @@ import * as webpackNumbers from 'webpack-numbers';
 webpackNumbers.wordToNum('Two');
 ```
 
-- __CommonJS module require:__
+- __CommonJS module requires:__
 
 ``` js
 const webpackNumbers = require('webpack-numbers');
 // ...
 webpackNumbers.wordToNum('Two');
 ```
 
-- __AMD module require:__
+- __AMD module requires:__
```

Comment:
that is a header for example `require` call of `AMD ` module, why is this change necessary?